src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%smooth-transition {
+  transition: $transitionDurating linear;
```

Comment:
Nope, don't fo it this way. You need to specify certain rules that you transition in each case.